document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImpl.java

```diff
@@ -1,12 +1,36 @@
 package uk.gov.companieshouse.document.generator.accounts;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.model.transaction.Transaction;
+import uk.gov.companieshouse.document.generator.accounts.service.TransactionService;
 import uk.gov.companieshouse.document.generator.interfaces.DocumentInfoService;
 import uk.gov.companieshouse.document.generator.interfaces.model.DocumentInfo;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
 
+@Service
 public class AccountsDocumentInfoServiceImpl implements DocumentInfoService {
 
+    @Autowired
+    private TransactionService transactionService;
+
+    public static final String MODULE_NAME_SPACE = "document-generator-accounts";
+
+    private static final Logger LOG = LoggerFactory.getLogger(MODULE_NAME_SPACE);
+
     @Override
     public DocumentInfo getDocumentInfo() {
-        return null;
+        LOG.info("Started getting document");
+
+        String resource = "";
+
+        Transaction transaction = transactionService.getTransaction(resource);
+        if (transaction == null) {
+            return null;
```

Comment:
Should we log if the transaction is null, to indicate that something has happened? Is returning null here safe?

Reply:
So this is part of the skeleton, it's still up in the air on what exactly I return. So I'll leave it null for the time being, however, I have missed out a logging line here to say the transaction is null, so I will add that :+1: